packages/circuit-ui/components/Badge/Badge.tsx

```diff
@@ -78,11 +72,8 @@ const variantStyles = ({
   variant = 'neutral',
 }: StyleProps & BadgeProps) => {
   const currentColor = COLOR_MAP[variant];
-  if (!currentColor) {
-    return null;
-  }
```

Comment:
This might technically be a breaking change, because if one implementation uses the `Badge` with a wrong variant name (e.g. `<Badge variant="error" />`—the right variant would be `danger`)

On the other hand, I think it might be better to let this fail in development, so that developers notice the mistake and can address it, rather than it failing silently.

Any app using TypeScript would not be impacted since build would have failed already if the variant name was wrong

Reply:
No strong opinions though, happy to revert this if we want to avoid the potential confusion for developers

Reply:
I believe this was a left-over from the static CSS extraction. Should be safe to remove.